config.js

```diff
@@ -17,10 +19,21 @@ exports.client = {
 }
 
 exports.github = {
+  'dev': {
+    'teamId': '1940407', // github.com/dvcoders-test
+    'apiKey': process.env.GITHUB_API_KEY_DEV || null
```

Comment:
I mean why do we have to separate GITHUB_API_KEY although we will not use the real api keys or tokens in development anyways.